sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -2009,6 +2023,10 @@ def get_page_ranges( # type: ignore
             The snapshot diff parameter that contains an opaque DateTime value that
             specifies a previous blob snapshot to be compared
             against a more recent snapshot or the current blob.
+        :keyword str managed_disk_diff:
+            Specifies the URL of a managed disk snapshot. The response will only
+            contain pages that were changed between the target blob and its previous
+            snapshot. Introduced in API version '2019-04-19'.
```

Comment:
This parameter name `managed_disk_diff` is just a place-holder for now - we will need to settle on a couple of parameter nameing/behaviour questions that I was to address separately from this PR.